runtime/binding-tcp/src/main/java/io/aklivity/zilla/runtime/binding/tcp/internal/TcpUsageTracker.java

```diff
@@ -21,14 +21,14 @@
 
 import io.aklivity.zilla.runtime.engine.EngineContext;
 
-public final class TcpCapacityTracker
+public final class TcpUsageTracker
 {
     private final int capacity;
     private final LongConsumer recordUsage;
 
-    private int usage;
+    private volatile int usage;
```

Comment:
Did we change this to `volatile` to handle access from engine boss thread during tcp accept?